### DISCUSSION TITLE
[Release Vote] Vote for Release V0.4.0-rc1

### DISCUSSION THREAD

**Discussion by zhoujinsong** [apache/amoro#656] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Arctic 0.4.0 release.

The commit ID is fbb85f1ca305041e739c833038e9398fb3c53342
* This corresponds to the tag: v0.4.0-rc1
* https://github.com/NetEase/arctic/commits/v0.4.0-rc1
* https://github.com/NetEase/arctic/tree/v0.4.0-rc1

Please check out,  build, and test.

[ ] +1 Release this as Arctic 0.4.0
[ ] +0
[ ] -1 Do not release this because...

Please vote in the next 72 hours and add your test steps when voting.

**Comment by zhoujinsong** [comment]
-1 

For testing the release in my local environment with the following step:

- Ran build with JDK8 and JDK11
- Optimize iceberg tables with format-version v1 and v2
- Check table watermark for arctic table

Found some bugs and improvements:

- #685 
- #689
- #692 
- #714

**Comment by majin1102** [comment]
default native iceberg optimizing has a poor MPP performance under lakehouse-benchmark, we shall give better default optimizing parameters for benchmark

**Comment by shidayang** [comment]
-1
For testing the release in my local environment with the following step:
- Docker benchmark
- Physical machine benchmark

Found some bugs and improvements:
- #711 
- #718

**Comment by Aireed** [comment]
-1 
For testing the release in my local environment with the following step:

- terminal  spark  ddl/dml
- flink quickstart in local machine
- dashboard  catalog  CRUD
- dashboard setting

Found some bugs:
- #724

**Comment by YesOrNo828** [comment]
-1 for now.
After testing this release(0.4.0) found the following bug:
#686
